R/data_to_wide.R

```diff
@@ -4,11 +4,18 @@
 #' the number of rows. This is a dependency-free base-R equivalent of
 #' `tidyr::pivot_wider()`.
 #'
-#' @param data A data frame to pivot.
-#' @param id_cols The name of the column that identifies the rows. If `NULL`,
-#' it will use all the unique rows.
-#' @param names_from The name of the column that contains the levels to be
-#' used as future column names.
+#' @param data A data frame to convert to wide format, so that it has more
+#' columns and fewer rows post-widening than pre-widening.
+#' @param by The name of the column that identifies the rows in the data by
+#' which observations are grouped and the gathered data is spread into new
+#' columns. Usually, this is a variable containing an ID for observations that
+#' have been repeatedly measured. If `NULL`, it will use all remaining columns
+#' that are not in `names_from` or `values_from` as ID columns. `by` can also
+#' be a character vector with more than one name of identifier columns. See also
+#' 'Details' and 'Examples'.
+#' @param names_from The name of the column in the original data whose values
```

Comment:
There can be several columns

Reply:
Yes, this is already mentioned in the docs:
> `by` can also be a character vector with more than one name of identifier columns.